app/src/main/java/com/ikarabulut/Game.java

```diff
@@ -3,24 +3,25 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class Evolve {
-    private Board board;
+public class Game {
+    private Board currentBoard;
     private Board nextBoard;
 
-    public Evolve(Board board) {
-        this.board = board;
-        this.nextBoard = new Board(board.getRows(), board.getColumns());
+    public Game(Board board) {
+        currentBoard = board;
+        nextBoard = new Board(board.getRows(), board.getColumns());
+
     }
 
     public List<ArrayList> getNextBoard() {
         return this.nextBoard.getBoard();
     }
 
     public void generateNextBoard() {
-        for (int row = 0; row < board.getRows(); row++) {
-            for (int column = 0; column < board.getColumns(); column++) {
-                int currentCellNumOfAliveNeighbors = board.getNumberOfAliveNeighbors(row, column);
-                boolean currentCellIsAlive = board.isCellAliveAt(row, column);
+        for (int row = 0; row < currentBoard.getRows(); row++) {
+            for (int column = 0; column < nextBoard.getColumns(); column++) {
```

Comment:
Just asking, there is any reason to use the rows from currentBoard and columns from nextBoard?

Reply:
Also, I like the name` Game` !

Reply:
I overlooked that, but it should have been changed. I updated this and also took Johns (@riccjohn) advice about generating new Cell objects instead of setting dead and alive. It should be in my most recent push